x/ecocredit/server/basket/put_test.go

```diff
@@ -352,6 +364,27 @@ func TestPut(t *testing.T) {
 			},
 			errMsg: "cannot put a credit from a batch with start date",
 		},
+		{
+			name:            "batch outside of years in the past",
+			startingBalance: "100000000",
+			msg: basket2.MsgPut{
+				Owner:       addr.String(),
+				BasketDenom: basketDenom3,
+				Credits:     []*basket2.BasketCredit{{BatchDenom: denom, Amount: "2"}},
+			},
+			expectedBasketCoins: "2000000", // 2 million
+			expectCalls: func() {
+				badTimeInfo := *batchInfoRes.Info
+				badYear := int(validYearsInThePast + 10)
+				badTime := time.Date(badYear, 1, 1, 0, 0, 0, 0, time.UTC)
+				badTimeInfo.StartDate = &badTime
+				ecocreditKeeper.EXPECT().
```

Comment:
i think this needs to be updated, the ORM updates broke this test a bit,, most things should be fixed by just adding `s.` or `s.stateStore.`